pages/vi/vi-team.md

```diff
@@ -35,6 +35,6 @@
 |[AlekseiKorolev](profiles/alekseikorolev.md)|2019-06-14|
 |[ArranHL](profiles/ArranHL.md)|2019-06-17|
 |[Chiragawale](profiles/chiragawale.md)|2019-06-17|
-
+|[Irisb1701](profiles/irisb1701.md)|2019-07-03|
```

Comment:
please put back empty line. It helps to separate between active and former member section

Reply:
Sorry about that, should be fixed now